File: app/src/main/java/cc/smartcash/wallet/Models/UserRegisterRequest.java
package cc.smartcash.wallet.Models;

import com.fasterxml.jackson.annotation.JsonProperty;

public class UserRegisterRequest {

    @JsonProperty("email")
    public String email;
    @JsonProperty("password")
    public String password;
    @JsonProperty("recoveryKey")
    public String recoveryKey;
    @JsonProperty("firstName")
    public String firstName;
    @JsonProperty("lastName")
    public String lastName;
    @JsonProperty("username")
    public String username;
    @JsonProperty("photo")
    public String photo;
    @JsonProperty("facebookId")
    public String facebookId;
    @JsonProperty("phone")
    public String phone;
    @JsonProperty("userDescription")
    public String userDescription;
    @JsonProperty("countryCode")
    public String countryCode;
    @JsonProperty("timeZone")
    public String timeZone;
    @JsonProperty("termsVersion")
    public String termsVersion;

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getRecoveryKey() {
        return recoveryKey;
    }

    public void setRecoveryKey(String recoveryKey) {
        this.recoveryKey = recoveryKey;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPhoto() {
        return photo;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }

    public String getFacebookId() {
        return facebookId;
    }

    public void setFacebookId(String facebookId) {
        this.facebookId = facebookId;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getUserDescription() {
        return userDescription;
    }

    public void setUserDescription(String userDescription) {
        this.userDescription = userDescription;
    }

    public String getCountryCode() {
        return countryCode;
    }

    public void setCountryCode(String countryCode) {
        this.countryCode = countryCode;
    }

    public String getTimeZone() {
        return timeZone;
    }

    public void setTimeZone(String timeZone) {
        this.timeZone = timeZone;
    }

    public String getTermsVersion() {
        return termsVersion;
    }

    public void setTermsVersion(String termsVersion) {
        this.termsVersion = termsVersion;
    }

}

File: app/src/main/java/cc/smartcash/wallet/Models/UserRegisterRequest.kt
package cc.smartcash.wallet.Models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
class UserRegisterRequest : Serializable {

    @JsonProperty("email")
    var email: String? = null

    @JsonProperty("password")
    var password: String? = null

    @JsonProperty("recoveryKey")
    var recoveryKey: String? = null

    @JsonProperty("firstName")
    var firstName: String? = null

    @JsonProperty("lastName")
    var lastName: String? = null

    @JsonProperty("username")
    var username: String? = null

    @JsonProperty("photo")
    var photo: String? = null

    @JsonProperty("facebookId")
    var facebookId: String? = null

    @JsonProperty("phone")
    var phone: String? = null

    @JsonProperty("userDescription")
    var userDescription: String? = null

    @JsonProperty("countryCode")
    var countryCode: String? = null

    @JsonProperty("timeZone")
    var timeZone: String? = null

    @JsonProperty("termsVersion")
    var termsVersion: String? = null

}



